packages/mui-material/test/typescript/moduleAugmentation/InputLabelCustomProps.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { InputLabel } from '@mui/material';
```

Comment:
Let's use path imports:
```suggestion
import InputLabel from '@mui/material/InputLabel';
```